tests/runtest.proj

```diff
@@ -71,7 +73,7 @@
 
     <Error Condition="'$(CLRTestPriorityToBuild)' == '0' and '$(TestCount)' &lt;= 2000" Text="Unexpected test count. Expected &gt; 2000, found $(TestCount).'" />
     <Error Condition="'$(CLRTestPriorityToBuild)' == '0' and '$(TestCount)' &gt;= 3000" Text="Unexpected test count. Expected &lt; 3000, found $(TestCount).'" />
-    <Error Condition="'$(CLRTestPriorityToBuild)' == '1' and '$(TestCount)' &lt;= 11500" Text="Unexpected test count. Expected &gt; 1150, found $(TestCount).'" />
+    <Error Condition="'$(CLRTestPriorityToBuild)' == '1' and '$(TestCount)' &lt;= 11000" Text="Unexpected test count. Expected &gt; 11000, found $(TestCount).'" />
```

Comment:
Long term we should find a better way to do this. Currently if we build pri1 tests, then build pri0 tests we will encounter an error. In addition if we build multiple different architectures or multiple build types we will also hit this error.

Reply:
I agree. Some comments:
1. The "build pri1 then pri0" or vice versa will give an error. That's kind of annoying. However, it's also weird that previously (though maybe not anymore?) if you did this, you would still *run* the pri1 set.
2. I don't think the multiple arch/build types will hit this, because they are in separate directories.

Reply:
I do not remember the specifics however, @briansull was hitting issues using build.cmd all related to having multiple different arch/build types getting this error.

Reply:
I believe you are correct, because we look over the directory to see what we will include in the xunit wrappers.

Reply:
Resolving as this should not block the pr.